puppet-blacksmith.gemspec

```diff
@@ -18,9 +18,10 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.4.0'
 
   s.add_runtime_dependency 'rest-client', '~>2.0'
+  s.add_development_dependency 'bundler'
   s.add_development_dependency 'rake'
-  s.add_development_dependency 'pdk'
-  s.add_development_dependency 'puppet', '>=2.7.16'
```

Comment:
Why drop the lower bound? Shouldn't hurt to have one.

Reply:
rolled back